modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
+* For Anypoint Studio 6.x and earlier, enable Maven support for your Mule project.
++
+See xref:6@studio::enabling-maven-support-for-a-studio-project.adoc[Enabling Maven Support for a Studio Project].
++
+All projects in Studio 7.x and later are mavenized by default.
 
-In combination with xref:alerts-on-runtime-manager.adoc[alerts], you can direct these notifications to different people to act on the alerts and fix the underlying problems.
 
-This example illustrates:
 
-* How to send business related events as notifications
-* How to send errors as notifications
-* How to send email alerts when notifications arrive
+== Custom Application Alerts
 
-[NOTE]
-CloudHub stores up to a maximum of 1000 notifications per application.
-If the number of notifications exceeds the maximum,
-CloudHub deletes the oldest notification.
-Each notification can contain up to 10,000 characters.
+Alerts enable you to send emails based on your application's notifications.
+You can create standard alerts in Runtime Manager for servers and applications deployed to either CloudHub or on-premises servers.
+See xref:alerts-on-runtime-manager.adoc[Alerts].
+
+
+You can also create _custom alerts_ for applications deployed to CloudHub.
+To create custom application alerts, you must add Anypoint Connector for CloudHub (CloudHub Connector)
+to flows in your Mule application.
+CloudHub Connector is available for both Mule 3 and Mule 4 applications.
 
-== Deploy Your Application
+=== Create a Custom Alert
 
-To deploy your application, in the Runtime Manager click *Deploy Application* from the *Applications* menu. For more information, see xref:deploying-to-cloudhub.adoc[Deploy to CloudHub].
+To create a custom alert in Anypoint Studio:
 
-== Trigger Your Notification
+. In Studio, add and configure CloudHub Connector in your Mule project:
++
+--
+.. In the Mule Palette view, click *CloudHub* and then drag the *Create Notification* operation to the application flow for which you want to trigger an alert in the canvas.
+.. In the canvas, click *Create Notification*.
+.. In the *Create Notification* properties window, click the green plus icon (*+*) to the right of the *Connector configuration* field.
+.. In the *Global Element Properties* window, configure authentication to Anypoint Platform and click *OK*:
++
+.The arrow shows the *Username* field in the *Global Element Properties* window.
+image::alerts-custom-ch-connector.png[Username field in the Global Element Properties window]
++
+The connector can use basic authentication or inherited token authentication.
+.. In the *Create Notification* properties window, enter the CloudHub app name in the *Domain* field and the notification in the *Message* field:
++
+.The arrow shows the *Message* field in the *Create Notification* properties window.
+image::alerts-custom-ch-connector-notification.png[Message field in the Create Notification properties window]
++
+For more information, see 
```

Comment:
GLOBALLY enhance cross-refs by making them specific: For more information <about what?>

Reply:
Seems redundant, but OK.
For more information about creating a notification with CloudHub Connector, see 
xref:connectors::cloudhub-connector/1.0/#creating-notifications[Creating a Notification].